.github/workflows/xpecRunner.yml

```diff
@@ -0,0 +1,47 @@
+# This is a basic workflow to help you get started with Actions
+
+name: runxpec
```

Comment:
It is XSpec, right? Also should we give it a friendly name [like the GSA/NIST devs do with OSCAL](https://github.com/18F/fedramp-automation/blob/master/.github/workflows/content-artifacts.yml#L13)? I was being old-school last time when I was worried about human-readable names and spaces, but it seems to work and make it more clear.

Should we make the intent a little more clear, like `OSCAL Validations: Unit Tests`?

Reply:
modified comments and job name to better reflect what we're doing